### ISSUE TITLE
Key outlines are invisible on e-paper

### ISSUE THREAD

**Issue by yodrew** [klausw/hackerskeyboard#809]
Thanks for your cool keyboard. 

I think the outlines of keys are supposed to be visible with your keyboard. However, they do not render on my e-paper tablet (Onyx Boox Max 3). A simple black and white theme might fix it.
![hackers-keyboard](https://user-images.githubusercontent.com/2936922/85622971-f36b3e80-b667-11ea-8749-e973ba13d0af.jpg)

**Comment by TPS** [comment]
Just _(non-constructively)_ commenting, I'm astonished HK displays as well as it does on e-Ink.  I can't imagine that such could've even been planned for at the time the underlying _(relatively ancient)_ codebase was established.

**Comment by yodrew** [comment]
Yes, it is very crisp. This e-paper has high resolution (225 DPI).

Oh, I should add that when I touch the keys, the grey box they are in appears.

**Comment by solroot** [comment]
#814
